docs/Release_notes.rst

```diff
@@ -3,12 +3,155 @@
+  * add csh -> tcsh symlink in recent tcsh easyconfigs (`#15571 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15571>`_)
+  * allow external tools to be located elsewhere for ETE (`#15578 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15578>`_)
+  * add additional sanity check commands for IQ-TREE v2.2.1 (`#15596 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15596>`_)
+* various bug fixes, including:
+  * fix source URL for freetype 2.6.5 with ``foss/2016b`` (`#14204 <https://github.com/easybuilders/easybuild-easyconfigs/pull/14204>`_)
+  * fix installation of easybuild-easyconfigs with setuptools>=61 by explicitly declaring there are no Python packages (`#15206 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15206>`_)
+  * use x.py to bootstrap Rust so that build options are properly passed through (`#15211 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15211>`_)
+  * fix RepeatMasker-4.1.2-p1 easyconfig by moving the database configure command to postinstallcmds (#15615, `#15615 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15615>`_)
+  * add hwloc dependency to tbb v2021.4.0 (`#15294 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15294>`_)
+  * tweak find command used in preconfigopts in easyconfig for pkg-config v0.29.2 with system toolchain to avoid descending into other filesystems (`#15313 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15313>`_)
+  * remove pkg-config use from SeqLib configure patch (avoids problem due to faulty autoconf macro) (`#15316 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15316>`_)
+  * update source URL for isl in GCCcore easyconfigs (`#15320 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15320>`_)
+  * update source URLs for YAXT 0.9.x to fix download (`#15323 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15323>`_)
+  * define ``$HHLIB`` as path to HH-suite installation directory, required by Perl scripts (`#15324 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15324>`_)
+  * add missing parallel and tbl2asn dependencies for prokka 1.14.5 (#15360, `#15381 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15381>`_)
+  * add missing dependencies for libheif (libpng, libjpeg-turbo) (`#15408 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15408>`_)
+  * switch to configuring build of libheif with CMake so libde265 dependency is picked up (`#15408 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15408>`_)
+  * disable use of ``-ftree-vectorize`` for OpenFOAM v2112 with ``foss/2021b`` (`#15495 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15495>`_)
+  * add patch for OpenMPI 4.1.1 to support building using ``--with-cuda=internal`` (#15528, `#15589 <https://github.com/easybuilders/easybuild-easyconfigs/pull/15589>`_)
```

Comment:
no link for `#15528` (there may be other "double" entries where a link is missing)